protocols/smtp/src/test/java/org/apache/james/protocols/smtp/netty/NettyStartTlsSMTPServerTest.java

```diff
@@ -98,48 +88,57 @@ private Protocol createProtocol(Optional<ProtocolHandler> handler) throws Wiring
 
     @Test
     public void connectShouldReturnTrueWhenConnecting() throws Exception {
-        ProtocolServer server = createServer(createProtocol(Optional.<ProtocolHandler> absent()), Encryption.createStartTls(BogusSslContextFactory.getServerContext()));  
-        server.bind();
-
+        ProtocolServer server = createServer(createProtocol(Optional.<ProtocolHandler> absent()), Encryption.createStartTls(BogusSslContextFactory.getServerContext()));
         SMTPSClient client = createClient();
-        InetSocketAddress bindedAddress = new ProtocolServerUtils(server).retrieveBindedAddress();
-        client.connect(bindedAddress.getAddress().getHostAddress(), bindedAddress.getPort());
-        assertThat(SMTPReply.isPositiveCompletion(client.getReplyCode())).isTrue();
 
-        client.quit();
-        client.disconnect();
+        try {
+            server.bind();
+            InetSocketAddress bindedAddress = new ProtocolServerUtils(server).retrieveBindedAddress();
+            client.connect(bindedAddress.getAddress().getHostAddress(), bindedAddress.getPort());
+            assertThat(SMTPReply.isPositiveCompletion(client.getReplyCode())).isTrue();
+        } finally {
+            client.quit();
+            client.disconnect();
+            server.unbind();
```

Comment:
Server is test depandant.
We could try to close after a null check in @after yes